food-bank-app/src/components/EventCard/EventButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTheme } from '@rneui/themed'
+import { TouchableOpacity, Text, GestureResponderEvent } from 'react-native'
+
+interface buttonProps {
+  onPress: (event: GestureResponderEvent) => void
+  title: string
+  rightborder?: boolean
+}
+
+const EventButton = ({ onPress, title, rightborder = false }: buttonProps): JSX.Element => {
+  const { theme } = useTheme()
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      style={{
+        flex: 1,
+        borderColor: theme.colors.buttonBorder,
+        borderTopWidth: 1,
+        borderRightWidth: rightborder ? 1 : 0,
+        alignSelf: 'stretch',
+        justifyContent: 'center'
+      }}>
+    <Text
+      style={{
+        color: theme.colors.green,
+        fontWeight: 'bold',
+        fontSize: 15,
+        textAlign: 'center'
+      }}>{title}</Text>
+    </TouchableOpacity>
+  )
+}
```

Comment:
para mantener consistencia, aquí podrías usar el stylesheet. Además, creo que faltan indentaciones
```suggestion
    <TouchableOpacity
      onPress={onPress}
      style={[styles.container, { borderColor: theme.colors.buttonRight, borderRightWidth: rightBorder }}>
      <Text
        style={[styles.title, { color: theme.colors.green }]}>
          {title}
        </Text>
    </TouchableOpacity>
  )
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    borderTopWidth: 1,
    alignSelf: 'stretch',
    justifyContent: 'center'
  },
  title: {
    fontWeight: 'bold',
    fontSize: 15,
    textAlign: 'center'
  }
})
```